object/init.go

```diff
@@ -291,3 +295,13 @@ func initBuiltInPermission() {
 	}
 	AddPermission(permission)
 }
+
+func InitLanguage() {
```

Comment:
Is this code and `github.com/beego/i18n` lib still useful for this PR? Can we delete it?